apps/condo/domains/acquiring/schema/CreatePaymentByLinkService.js

```diff
@@ -0,0 +1,250 @@
+/**
+ * Generated by `createservice acquiring.CreatePaymentByLinkService`
+ */
+
+const { get } = require('lodash')
+
+const { createInstance } = require('@open-condo/clients/address-service-client')
+const { GQLError, GQLErrorCode: { BAD_USER_INPUT } } = require('@open-condo/keystone/errors')
+const { GQLCustomSchema, getById } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/acquiring/access/CreatePaymentByLinkService')
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/acquiring/constants/context')
+const {
+    registerMultiPaymentForOneReceipt,
+    registerMultiPaymentForVirtualReceipt,
+} = require('@condo/domains/acquiring/utils/serverSchema')
+const { AcquiringIntegrationContext } = require('@condo/domains/acquiring/utils/serverSchema')
+const { BankAccount } = require('@condo/domains/banking/utils/serverSchema')
+const {
+    validateQRCode,
+    BillingReceipt,
+    BillingIntegrationOrganizationContext,
+    BillingRecipient,
+} = require('@condo/domains/billing/utils/serverSchema')
+const { WRONG_FORMAT } = require('@condo/domains/common/constants/errors')
+const { Property } = require('@condo/domains/property/utils/serverSchema')
+
+/**
+ * List of possible errors, that this custom schema can throw
+ * They will be rendered in documentation section in GraphiQL for this custom schema
+ */
+const ERRORS = {
+    ADDRESS_IS_INVALID: {
+        mutation: 'createPaymentByLink',
+        code: BAD_USER_INPUT,
+        type: WRONG_FORMAT,
+        message: 'The provided address is invalid',
+    },
+    BANK_ACCOUNT_IS_INVALID: {
+        mutation: 'createPaymentByLink',
+        code: BAD_USER_INPUT,
+        type: WRONG_FORMAT,
+        message: 'Provided bank account is not in the system',
+    },
+}
+
+const CreatePaymentByLinkService = new GQLCustomSchema('CreatePaymentByLinkService', {
+    types: [
+        {
+            access: true,
+            type: 'input CreatePaymentByLinkInput { dv: Int!, sender: SenderFieldInput!, qrCode: String! }',
+        },
+        {
+            access: true,
+            type: 'type CreatePaymentByLinkOutput { multiPaymentId: ID!, address: String!, unitName: String!, accountNumber: String! }',
+        },
+    ],
+
+    mutations: [
+        {
+            access: access.canCreatePaymentByLink,
+            schema: 'createPaymentByLink(data: CreatePaymentByLinkInput!): CreatePaymentByLinkOutput',
+            resolver: async (parent, args, context) => {
+                const { data: { dv, sender, qrCode } } = args
+
+                // Stage 0: validate QR code and normalize address
+                const validationResult = await validateQRCode(context, { dv, sender, qrCode })
+
+                const {
+                    qrCodeFields: {
+                        PersonalAcc, // organization's bank account
+                        BIC,
+                        PayerAddress,
+                        PaymPeriod, // mm.yyyy
+                        Sum,
```

Comment:
you need to validate amount as it can contain spaces and coma

Reply:
It looks like this string should not contain any punctual symbols
<img width="1280" alt="image" src="https://github.com/open-condo-software/condo/assets/25384290/10c31343-5cf0-46a8-b1e9-fcfced49dc5a">

Reply:
According to this 👆 I've divided the sum by 100.